### PR TITLE
BETA: Register riddim_glitch.is-a.dev

### PR DESCRIPTION
**Pull request by GH0STP4WZ**
Added `riddim_glitch.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Riddim-GLiTCH** [comment]
Bump

File: domains/riddim_glitch.json
Comment (CuteDog5695) [inline domains/riddim_glitch.json:1]: i dont think we allow domains with underscores

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> Should probably explain the funny screenshot, I previously registered for a domain which now no longer is under my list of domains, this PR is just me trying to re-add it. Though I think I originally had it registered as "riddim-glitch" instead of "riddim_glitch".

checked and i see that you originally registered riddim-glitch.is-a.dev

**Comment by Riddim-GLiTCH** [comment]
> > Should probably explain the funny screenshot, I previously registered for a domain which now no longer is under my list of domains, this PR is just me trying to re-add it. Though I think I originally had it registered as "riddim-glitch" instead of "riddim_glitch".
> 
> checked and i see that you originally registered riddim-glitch.is-a.dev

Indeed, however it does not appear on my dashboard, and visiting it takes you to your homepage. this is why I registered again.

**Comment by CuteDog5695** [comment]
> > > Should probably explain the funny screenshot, I previously registered for a domain which now no longer is under my list of domains, this PR is just me trying to re-add it. Though I think I originally had it registered as "riddim-glitch" instead of "riddim_glitch".
> > 
> > 
> > checked and i see that you originally registered riddim-glitch.is-a.dev
> 
> Indeed, however it does not appear on my dashboard, and visiting it takes you to your homepage. this is why I registered again.

think it got caught in cleanup, btw rename riddim_glitch.json with riddim-glitch.json and then i can merge PR

**Comment by Riddim-GLiTCH** [comment]
Renamed the files. My old domain was still present so I had to remove it in order to be able to save the file.

**Review by iostpa — DISMISSED** [review]
seems fine to me now

**Comment by CuteDog5695** [comment]
i dont know if you noticed but this PR has a merge conflict, reason why i didnt merge PR. please fix the merge conflict problem and then ill merge it

**Comment by CuteDog5695** [comment]
> I have noticed but I'm not sure which file is conflicting. I'm also not sure where to start looking, to be honest.

you can try and google it

**Comment by Riddim-GLiTCH** [comment]
Hi, apologies for the long delay in response. I was dealing with family matters and it consumed most of my free time, resulting in me not being able to check my github.

I just looked up the issue, but unless im misunderstanding the actual issue, it should already be solved?
All I could find was a potential lack of entries in the record file, Which should be mitigated with at least one entry, which *should* be present in the form of a CNAME. Unless the issue is the mismatching email compared the email in the record for my website, which I just updated.

**Comment by CuteDog5695** [comment]
> Hi, apologies for the long delay in response. I was dealing with family matters and it consumed most of my free time, resulting in me not being able to check my github.
> 
> I just looked up the issue, but unless im misunderstanding the actual issue, it should already be solved? All I could find was a potential lack of entries in the record file, Which should be mitigated with at least one entry, which _should_ be present in the form of a CNAME. Unless the issue is the mismatching email compared the email in the record for my website, which I just updated.

merge conflict seems to still be here, ill list a couple tutorials here:
https://www.freecodecamp.org/news/how-to-fix-merge-conflicts-in-git/
https://www.youtube.com/watch?v=mSfq1SoMocg
https://stackoverflow.com/questions/161813/how-do-i-resolve-merge-conflicts-in-a-git-repository

**Comment by Riddim-GLiTCH** [comment]
Thanks, I will check these out tomorrow when I am done with work.

**Comment by 21Z** [comment]
Your fork is outdated, `random-glitch.json` doesnt exist on this repo, but ur fork does...
Sync your fork and create a new file `random-glitch.json`

**Comment by TerraPlayz** [comment]
Updates??

**Comment by Riddim-GLiTCH** [comment]
> > My old domain was still present so I had to remove it
> 
> Your fork is outdated, Sync your fork and create a new file `random-glitch.json`

Done this

**Review by Stef-DP — APPROVED** [review]
seems good to me now

**Pull request merged**